.github/workflows-source/ci-main.yml

```diff
@@ -300,12 +300,24 @@ jobs:
           # Setup zig-cache
           mkdir -p /tmp/zig-cache
 
+          bazel_targets=(
+            # make sure codebase builds for local development
+            //rs/...
+            //publish/binaries/...
+
+            # necessary artifacts & tests for pocket -ic
+            //packages/pocket-ic:all \
+            //rs/pocket_ic_server:test \
+            //rs/pocket_ic_server:gateway \
+            //rs/pocket_ic_server:pocket-ic-server
```

Comment:
Aren't these paths subsumed in `//rs/...`?

Reply:
It makes sense to be explicit. Could you please leave a comment about the intent of repeating the paths?